### PR TITLE
Update the default value of log-file for cdc

### LINKED ISSUE #4450: Update the default value of log-file for cdc
## Bug Report

**What version of Kubernetes are you using?**

**What version of TiDB Operator are you using?**

**What storage classes exist in the Kubernetes cluster and what are used for PD/TiKV pods?**

**What's the status of the TiDB cluster pods?**

**What did you do?**

**What did you expect to see?**
Currently, we set `/dev/stderr` as the default value for the log-file of cdc, however, it seems that TiCDC takes it as a regular file and write to the root dir of the container:
middle_img_v2_9fffac76-e1c6-4839-bd1e-e2d7a9b020dg
We need to set the default value to "".
**What did you see instead?**

### PR DESCRIPTION
**Pull request by KanShiori**
### What problem does this PR solve?

Close #4450
### Code changes

- [x] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [x] Manual test 

- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
Update the default value of log-file for cdc to avoid overwriting `/dev/stderr`, it will cause rolling upgrade if not set `log-file` in config
```

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]

**Review by july2993 — APPROVED** [review]

**Review by csuzhangxc — APPROVED** [review]

File: tests/e2e/tidbcluster/serial.go [hunk 1]
```diff
@@ -846,7 +846,11 @@ var _ = ginkgo.Describe("[Serial]", func() {
 
 				ginkgo.By("Deploy original TiDB cluster")
 				utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc, 10*time.Minute, 5*time.Second)
-				selector := MustGetLabelSelectorForComponents(tcName, label.DiscoveryLabelVal, label.PumpLabelVal) // ingore discovery and pump
+				selector := MustGetLabelSelectorForComponents(tcName,
+					label.DiscoveryLabelVal,
+					label.PumpLabelVal,
+					label.TiCDCLabelVal, // ingore ticdc because of PR #4494
```
Comment (DanielZhangQD) [inline tests/e2e/tidbcluster/serial.go:852]: Do we still need to add this as we have updated the code in `tests/pkg/fixture/fixture.go`?

**Pull request merged**